utils/metrics/metrics_util.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	metrics "github.com/armon/go-metrics"
 	"github.com/cosmos/cosmos-sdk/telemetry"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
Do you know if `metrics "github.com/armon/go-metrics"` works here too? telemetry in cosmos-sdk uses this go-metrics already so it would be nice to not introduce any more dependencies

Reply:
Yeah also tried it's in the second last commit and it didn't work, which is pretty confusing

Reply:
gotcha, that's unfortunate since it looks like we lose some of the global labels like chain id

Reply:
Good point, I'm trying to append `globalLabels` from telemetry by accessing `telemetry.globalLabels` but somehow it isn't working, do you know why it can't just be accessed like that? The file also has `package telemetry` declared so I'm assuming it should work
https://sourcegraph.com/github.com/cosmos/cosmos-sdk@c24c439/-/blob/telemetry/metrics.go?L17